docs/user_docs/query_docs/HYPERVISORS.md

```diff
@@ -43,6 +43,7 @@ from enums.query.props.hypervisor_properties import HypervisorProperties
 | HYPERVISOR_VCPUS              | `int`    | "vcpus"                            | The number of vCPUs on this hypervisor.                         |
 | HYPERVISOR_VCPUS_USED         | `int`    | "vcpus_used"                       | The number of vCPUs currently being used on this hypervisor.    |
 | HYPERVISOR_DISABLED_REASON    | `string` | "disabled_reason"                  | Comment of why the hypervisor is disabled, None if not disabled |
+| HYPERVISOR_UPTIME             | `string` | "uptime"                           | The total uptime of the hypervisor and info about average load  |
```

Comment:
Should this one property be holding 2 pieces of information? Also, how can it hold 2 pieces of information?

Reply:
what about HYPERVISOR_UPTIME and HYPERVISOR_AVERAGE_LOAD? 
I agree with @khalford 
More variables is easier to understand that less but more complex ones.

Reply:
The openstack API response returns this all in a single string: 
`"uptime": " 08:32:11 up 93 days, 18:25, 12 users,  load average: 0.20, 0.12, 0.14"` - [source](https://docs.openstack.org/api-ref/compute/#list-hypervisors-details)

I am not sure if we should be re-formatting the API responses should be done here as the since the other properties match the API response. @anish-mudaraddi any thoughts?

Reply:
Ah, so it is just the standard output of the UNIX command "uptime"...

Reply:
Another minor thing, @gmatthews20 
I see the commit message explains what has changed in this PR, but not why. Unless it is self-explanatory, it is always a good practice to explain why a change is being done.

Reply:
🤔 why to import "re" if it is not being used?

Reply:
I think your change works 👍